README.md

```diff
@@ -16,6 +16,22 @@ It's possible to implement a custom handling of incoming messages, but the defau
 To implement your own custom handling of received messages, override the operation `TCPServer::handleReceivedMessageCustom()` and set the config property `defaultHandlingOfReceivedMessages` to false.
 
 ## Build the Library
+
+### Master branch
+The latest version of library removed the POCO dependency and using APIs from TargetRTS.
+
+To make use of this library add the TC `tcpServerLib.tcjs` as a prerequisite of the executable TC for your Model RealTime application.
+
+Note that the TC `tcpServerLib.tcjs` is designed to be built as a prerequisite of an executable TC and will reuse some of the TC properties from that executable (using the topTC variable). Here is an example of such a TC property:
+
+`
+tc.compileArguments = topTC.compileArguments || '$(DEBUG_TAG)';
+`
+
+If you want to be able to build the library TC directly (i.e. as a "top TC"), you should first modify these TC properties (essentially making sure that the value after the '||' is appropriate).
+
+### Tag V1.0
+The release Tag V1.0 of this library depends on POCO library and below instruction must be follow to use this release.
```

Comment:
You don't need to document how to use the Poco version, since that's already documented in the README the user will see when he checks out that branch.